Functions/xASL_test_Flavors.m

```diff
@@ -1,26 +1,27 @@
-function xASL_test_Flavors(pathExploreASL, pathTest, bTest, x)
+function flavors = xASL_test_Flavors(testConfig, bTest, x, flavors)
```

Comment:
`flavors` seems to be optional. But if I don't provide it, I get an error:

`Not enough input arguments.
Error in xASL_test_Flavors (line 74)
        flavors.loggingTable = xASL_test_Flavors_DCM2BIDS(testConfig, x, flavors.loggingTable);`

And it's not really clarified what should I input...

Reply:
Should be optional but really *recommended* right now 04b6f5a 👍